### PR TITLE
changes made for internal access, template detail api change

### PR DESCRIPTION
**Pull request by VISHNUDAS-tunerlabs**
changes made for templateDetail api access

### CODE REVIEW

File: config/db/mongodb.js [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 
 //dependencies
+//require('dotenv').config();
```
Comment (priyanka-TL) [inline config/db/mongodb.js:10]: @VISHNUDAS-tunerlabs remove this

File: controllers/v1/solutions.js [hunk 2]
```diff
@@ -985,4 +985,54 @@ module.exports = class Solutions extends Abstract {
...
+   * @api {get} /kendra/api/v1/solutions/getList
+   * Lists of solution types.
+   * @apiVersion 0.0.1
+   * @apiName Lists of solution.
+   * @apiGroup Solutions
+   * @apiHeader {String} X-authenticated-user-token Authenticity token
+   * @apiSampleRequest /kendra/api/v1/solutions/getList
+   * @apiUse successBody
+   * @apiUse errorBody
+   * @apiParamExample {json} Response:
```
Comment (priyanka-TL) [inline controllers/v1/solutions.js:998]: @VISHNUDAS-tunerlabs add  the proper response here

File: generics/middleware/authenticator.js [hunk 3]
```diff
@@ -79,6 +80,7 @@ module.exports = async function (req, res, next) {
       performInternalAccessTokenCheck = true;
     }
   }));
+  console.log(token,performInternalAccessTokenCheck,process.env.INTERNAL_ACCESS_TOKEN)
```
Comment (priyanka-TL) [inline generics/middleware/authenticator.js:83]: @VISHNUDAS-tunerlabs remove console log

**Comment by aks30** [comment]
Not required.

**Pull request closed without merging**